Added RightSidebar Kodi icon and  hover effect on progress slider

developingright opened the pull request:
Hi @razzeee ,
I have added Kodi icon to the RightSidebar and hover effect on the progress slider like in Chorus-2.

Name             |  RideSidebar Kodi button  |  Progress Slider
:-------------------------:|:-------------------------: |:-------------------------:
Chorus-2 | ![Screenshot from 2022-04-02 14-50-39](https://user-images.githubusercontent.com/18377109/161377560-85f18cb0-3c89-427d-93bc-846eccc8bc6e.png) | ![Screenshot from 2022-04-02 14-59-54](https://user-images.githubusercontent.com/18377109/161377623-099479bb-4c57-47fe-910c-7ba2fb293d71.png)

****
Name             |  Before making changes  |  After making changes
:-------------------------:|:-------------------------: |:-------------------------:
Elm-chorus -> RightSidebar Kodi button  |  ![161093468-caf63bb8-6d88-42d6-9b6a-e405bcc38f67 (1)](https://user-images.githubusercontent.com/18377109/161377652-3c7b8b56-7de0-4e9a-b397-587addca0a12.png) |  ![Screenshot from 2022-04-02 14-52-23](https://user-images.githubusercontent.com/18377109/161377806-900460ca-36d6-40de-a2cd-a46f9fc8cce1.png)
Elm-Chorus -> Progress Slider | ![Screenshot from 2022-04-02 15-05-56](https://user-images.githubusercontent.com/18377109/161377689-856bab5e-772b-4d1a-925a-5287057c1e73.png) | ![Screenshot from 2022-04-02 15-08-02](https://user-images.githubusercontent.com/18377109/161377741-40c2465c-b112-4e56-a321-ca65bec3336d.png)

developingright commented on src/Components/RightSidebar.elm:
```diff
@@ -17,7 +18,17 @@ view { showRightSidebarMenu, showRightSidebarMenuMsg, clearPlaylistMsg, refreshP
+                        Element.row []
+                            [ image [ width (px 15), height (px 15), paddingXY 15 0 ]
+                                { description = ""
+                                , src = "https://raw.githubusercontent.com/ionif/elm-chorus/master/logo.png"
```
Hi @razzeee , 
	I have made changes to the code, now the images which is being used in the rightsidebar is local.

developingright commented:
@razzeee ,
Since Elm doesn’t support autoloading, one way to achieve this was to use the development server to serve the static files (for ex: “http:localhost:1234/images/logo.png” as image src). Since we are using parcel for running the development server, I have installed a parcel plugin “[parcel-plugin-static-files-copy](https://www.npmjs.com/package/parcel-plugin-static-files-copy)” and created an image folder in public folder which is mapped to the server. Now, I will create a PR to make all of the static files link local which is an already existing [issue](https://github.com/xbmc/elm-chorus/issues/36).

razzeee commented:
This should be a dev dependency, not a dependency.

It also seems to be a parcel 1 plugin, but we're using parcel 2

developingright commented:
Hi @razzeee,
I have updated the package.json file and updated the plugin which uses parcel2 “[parcel-reporter-static-files-copy](https://github.com/elwin013/parcel-reporter-static-files-copy)” for serving the static files.

razzeee commented:
This is conflicting now, you probably need to rebase

developingright commented:
Hi @razzeee,
I have rebased the code and resolved the conflicts.

razzeee commented:
I don't think you did that correctly :)
There are a bunch of unrelated commits
